docker-compose.yaml

```diff
@@ -14,45 +14,62 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+x-image:
+   &image
+   image: ${OZONE_IMAGE:-apache/ozone}:${OZONE_IMAGE_VERSION:-1.4.1}${OZONE_IMAGE_FLAVOR:--rocky}
+
+x-common-config:
+   &common-config
+   OZONE-SITE.XML_hdds.datanode.dir: "/data/hdds"
+   OZONE-SITE.XML_ozone.metadata.dirs: "/data/metadata"
+   OZONE-SITE.XML_ozone.om.address: "om"
+   OZONE-SITE.XML_ozone.om.http-address: "om:9874"
+   OZONE-SITE.XML_ozone.recon.address: "recon:9891"
+   OZONE-SITE.XML_ozone.recon.db.dir: "/data/metadata/recon"
```

Comment:
Is this config necessary? It looks like the intent is for all other metadata configs to fall back to `ozone.metadata.dirs`

Reply:
Probably not necessary, but falling back to `ozone.metadata.dirs` produces warning in the log.

And it was present in `docker-config`, I just moved and sorted the properties.